vscode/src/core/saros-extension.ts

```diff
@@ -0,0 +1,108 @@
+import { ExtensionContext, workspace, window } from "vscode";
+import { SarosServer } from "./saros-server";
```

Comment:
We should think about different names for the LSP saros server and the saros server.

Reply:
Second this! Made a proposal in the new commit, please have a look. At least it makes it more obvious that lsp server is meant.